lib/custatevec/Project.toml

```diff
@@ -11,7 +11,7 @@ cuQuantum_jll = "b75408ef-6fdf-5d74-b65a-7df000ad18e6"
 
 [compat]
 CEnum = "0.2, 0.3, 0.4, 0.5"
-CUDA = "~5.1, ~5.2"
```

Comment:
cuQuantum subpackages don't support CUDA 5.3 yet, as that pulls in cuTENSOR 2.0.